core/src/main/resources/reference.conf

```diff
@@ -269,7 +269,12 @@ base {
   akka.scheduler.tick-duration = 1
 
   akka {
-    
+
+    test {
+      ##  duration to wait in expectMsg in akka test
+      single-expect-default = 10s
+    }
+
```

Comment:
@manuzhang 

We need to make this config global visible. 
When user create a default ActorSystem, then test.conf will not be loaded.